.gitignore

```diff
@@ -11,3 +11,14 @@ __pycache__
 # IDEs
 .vscode
 compile_commands.json
+
+# Downloaded FVPs
+/fvp/AEMv8R_base_pkg/
```

Comment:
To make this easier to maintain in the long run, would it better to move the FVP's install directory to live inside the build directory instead?
That way we decouple it from the source tree, and make it simpler for developers to use their own out-of-tree FVP installations if they wish to.

Reply:
There is already an `FVP_INSTALL_DIR` cmake option to point it at an existing copy of the FVPs, but I've just realised that I'm also accessing the config files using that, so I'll re-arrange things to have two directories. However, I think leaving it in the source tree by default makes it simpler to get running, because `get_fvps.sh` knows where the source directory is, but not the install directory.

Reply:
Thanks! In that case, could we have a new directory under `fvp` to hold all of the downloaded FVPs (e.g. `fvp/install/`)? That way we can gitignore this entire directory and don't have to maintain this list of individual FVPs going forwards.

Reply:
That's exactly what the latest patch does:
* `fvp/download/` holds the downloaded files, is only used by `get_fvps.sh`, and is in .gitignore
* `fvp/install/` holds the installed FVPs, this can be moved using the `FVP_INSTALL_DIR` cmake option, and is in .gitignore
* `fvp/config/` holds the config files, and is normal source code, not in .gitignore, and accessed relative to `CMAKE_CURRENT_SOURCE_DIR` so it doesn't need to be copied to an external FVP installation

Reply:
Oh, brilliant! Thanks! Sorry I missed the patch updates in the PR.